store/sqlstore/team_store.go

```diff
@@ -1153,6 +1153,9 @@ func (s SqlTeamStore) GetTeamsForUser(userId string) ([]*model.TeamMember, *mode
 	return dbMembers.ToModel(), nil
 }
 
+// GetTeamsForUserWithPagination returns limited TeamMembers according to the perPage parameter specified.
+// It also offsets the records as per the page parameter supplied.
+// Returns an error if an error occurs during querying.
```

Comment:
I think this line doesn't say much here, and we are not providing this text anywhere else, I would just omit this line. What do you think @justinegeffen?